examples/src/vsock/client.rs

```diff
@@ -0,0 +1,47 @@
+#![cfg_attr(not(unix), allow(unused_imports))]
+
+pub mod hello_world {
+    tonic::include_proto!("helloworld");
+}
+
+use hello_world::{greeter_client::GreeterClient, HelloRequest};
+use tokio_vsock::VsockStream;
+use tonic::transport::{Endpoint, Uri};
+use tower::service_fn;
+
+// Use vsock-loopback so we don't need to spin up a VM. Must match server.
+static TEST_CID: u32 = vsock::VMADDR_CID_LOCAL;
+// Arbitrarily choosen. Must match server.
+static TEST_PORT: u32 = 8000;
+
+// Virtio VSOCK does not use URIs, hence this URI will never be used.
+// It is defined purely since in order to create a channel, since a URI has to
+// be supplied to create an `Endpoint`.
+static IGNORED_ENDPOINT_URI: &str = "file://[::]:0";
```

Comment:
Not proud of this, but this matches the code of the UDS example. 

In the long term it would probably be a good idea to update the endpoint struct to no longer require a URI. However that should be done in a separate PR, which also updates the UDS code.